packages/mobile/src/send/SendConfirmation.tsx

```diff
@@ -55,6 +55,11 @@ import { fetchDollarBalance } from 'src/stableToken/actions'
 import Logger from 'src/utils/Logger'
 import { currentAccountSelector, isDekRegisteredSelector } from 'src/web3/selectors'
 
+export interface CurrencyInfo {
+  currencyCode: LocalCurrencyCode
+  exchangeRate?: string | null
```

Comment:
When is exchange rate `null` or `undefined`? It looks like `fetchExchangeRate` async function always resolves to a string?

Reply:
in the case that we're not navigating to `SendConfirmation` from `handleSendPaymentData` (ie. from `SendAmount` maybe), the `CurrencyInfo` variable is undefined, and the `localCurrencyExchangeRate` is set by `getLocalCurrencyExchangeRate(state)` instead, which could be null. So Typescript throws an error if I don't include null here

Reply:
If exchangeRate is only passed sometimes, could it make sense to be passing currencyCode and exchangeRate directly rather than as a CurrencyInfo object to make it clear when exchangeRate is included or not?

Reply:
exchangeRate is always passed, resolving